auth_integration_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build integration
```

Comment:
Do we need to keep those ?

Reply:
You are right, probably we can delete it, since we don't use `go build`